frontend/src/pages/process.tsx

```diff
@@ -31,7 +32,7 @@ const Process: NextPage = () => {
         />
       </FullWidthAlert>
       <Breadcrumbs breadcrumbList={PROCESS_CRUMBS} />
-      Process Placeholder
+      <ProcessContent />
```

Comment:
Never mind. Duh, There are multiple sections of the page. This splits them out. I was confused because of the name of `ProcessContent`, thinking it was the content for the whole Process page.